dash_html.py

```diff
@@ -38,14 +38,20 @@
     CLASSICAL_TAB_LABEL,
     DESCRIPTION,
     DWAVE_TAB_LABEL,
-    SHOW_CQM,
     MAIN_HEADER,
     SCENARIOS,
+    SHOW_CQM,
     SOLVER_TIME,
-    THUMBNAIL
+    THEME_COLOR_SECONDARY,
+    THUMBNAIL,
 )
+from src.job_shop_scheduler import HybridSamplerType, SamplerType
 
-SOLVER_OPTIONS = ["Quantum Hybrid (CQM)", "Quantum Hybrid (NL)", "Classical (HiGHS)"]
+SAMPLER_TYPES = {
+    SamplerType.HYBRID: "Quantum Hybrid" if SHOW_CQM else "Quantum Hybrid (NL)",
+    SamplerType.HIGHS: "Classical (HiGHS)",
+}
+HYBRID_SAMPLER_TYPES = {HybridSamplerType.NL: "NL", HybridSamplerType.CQM: "CQM"}
```

Comment:
(minor) Aesthetically, I think `NL` and `CQM` looks a bit short in the dropdown. Perhaps use the full names here instead?
```suggestion
HYBRID_SAMPLER_TYPES = {HybridSamplerType.NL: "Nonlinear (NL)", HybridSamplerType.CQM: "Constraint Quadratic Model (CQM"}
```